roles/linux/dashboards/tasks/main.yml

```diff
@@ -8,6 +8,8 @@
   selinux:
     state: disabled
   when: (ansible_distribution != "Ubuntu") and (ansible_distribution != "Amazon")
```

Comment:
@heiderich Initially I added `when` condition to do this task only for OS distribution other than ubuntu and amazon linux where I faced different errors due to unavailable of this module in those distributions.

If this [condition](https://github.com/opensearch-project/ansible-playbook/pull/122/files#diff-26da4e2fc3ae1d719ad41b3e625de34dd3fbc83814e8a4c6875c55f4fb6f0e91R12) covers errors from `Ubuntu` and `amazon` distributions then we can remove `when` completely.

Reply:
@heiderich could you please check the above comments?